packages/dev/test/common/StatisticsManager.test.ts

```diff
@@ -46,11 +46,11 @@ contract('StatisticsManager', function (accounts) {
     await env.relayServer.createRelayTransaction(await env.createRelayHttpRequest())
     await env.relayServer.createRelayTransaction(await env.createRelayHttpRequest())
     await env.relayServer.createRelayTransaction(await env.createRelayHttpRequest())
+    await misbehavingPaymaster.setRevertPreRelayCallOnEvenBlocks(true)
     block = await web3.eth.getBlockNumber()
     if (block % 2 !== 0) {
       await evmMine()
     }
```

Comment:
how does this help? :-)

Reply:
That's a difference between ganahce and hardhat, setting revert on even blocks is advancing the block number as well.

Reply:
wait, every transaction increases block number on both hardhat and ganache, doesn't it?